### PR TITLE
K8s yaml chaos

### PR DESCRIPTION
**Pull request by rynewang**
Adds a chaos test with injected network delays. The test set up is a KubeRay cluster in a kind cluster, with Chaos Mesh to inject faults. Due to the size of the test machine we have 1 head node + 2 worker nodes. In this initial commit we have a network delay test: all networks are injected 200ms delay to simulate cross-continent traffic. 

Adds a shell script to install Helm.

The Ray job under test is a group of 3 Actors sending messages in a round-robin way for 1000 messages. First we run it without faults injected to make sure the Ray cluster itself works; then we inject the 200ms delay and rerun the job. Example run is https://buildkite.com/ray-project/oss-ci-build-pr/builds/31681#0189e82c-e90e-4f8f-aef3-e3a4c18866e1 where the no fault test ran for 25s, while the +delay test ran for 2m51, printed errors but passed.

### CODE REVIEW

File: python/ray/tests/chaos/chaos_network_bandwidth.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  mode: all
+  selector:
+    namespaces:
+      - default
+    labelSelectors:
+      'ray.io/cluster': 'raycluster-kuberay' # inject to all pods
+  bandwidth:
+    rate: '1mbps'
+    limit: 20971520
+    buffer: 10000
```
Comment (rkooo567) [inline python/ray/tests/chaos/chaos_network_bandwidth.yaml:17]: newline?

File: python/ray/tests/chaos/potato_passer.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import asyncio
+import argparse
+import ray
+ray.init()
+
+"""
+Potato passer is a test script that lets multiple actors call each other's methods.
```
Comment (rkooo567) [inline python/ray/tests/chaos/potato_passer.py:7]: I like the name lol

File: python/ray/tests/chaos/runtime_env.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+env_vars:
+  RAY_DEDUP_LOGS: "0"
```
Comment (rkooo567) [inline python/ray/tests/chaos/runtime_env.yaml:2]: newline!

**Review by rkooo567 — APPROVED** [review]

**Comment by rynewang** [comment]
lint fixed

**Comment by rkooo567** [comment]
can you ping me when it is ready to be merged?

**Pull request merged**